presto-main/src/main/java/com/facebook/presto/execution/Output.java

```diff
@@ -59,6 +65,12 @@ public String getTable()
         return table;
     }
 
+    @JsonProperty
+    public List<DateTime> getLastDataCommitTimes()
```

Comment:
I joda DateTime jackson serializable ?

Reply:
This is the key issue that we would like to resolve in this task. When I used just integers, it works fine. But when we start to use datetime, we start to see serde issues. The approach that we take is to split the diffs into multiple PRs, and test them gradually. So far I tested in verifier, it worked. Datetime does cause serde issues when we put the lastaccesstime into Table and Partition classes. Therefore, I will use integer instead. You will see that in the next PR.